### ISSUE TITLE
[Data Addition]: Add total corners and and total length to circuit data

### ISSUE THREAD

**Issue by MrBiscuit921** [jolpica/jolpica-f1#123]
this would be very helpful, i have no clue how hard this would be to add, if its possible im happy to enter in the data myself, but i would just really like this

**Comment by jolpica** [comment]
This is something we'd consider when we start creating new endpoints and move away from the ergast compatible endpoints (probably sometime in the next season). But theres no reason why we can't start working on data sources for this sooner.

What sources for this data are available?

**Comment by harningle** [comment]
[Event notes](https://www.fia.com/sites/default/files/decision-document/2024%20Austrian%20Grand%20Prix%20-%20Event%20Notes%20-%20Circuit%20Map,%20Pit%20Lane%20Drawing%20and%20Red%20Zones.pdf) are a solid source

I don't think it's easy to extract data from this PDF. Maybe it's going to be a manual work, or we should find a better source

**Comment by theOehrly** [comment]
Some notes on this, corner numbers can have additional letters appended. For example, Hungary has corner "1" and "1A" as well as "12" and "12A". This might be relevant for parsing data sources or for storing the numbers themselves, if we wanted to do such a thing.

Also, trying to count corners based on track curvature using telemetry data as input source does not work. People have tried this extensively. But there is no real consistency between what counts as a corner or what doesn't.

Therefore, we need some sort of official document that includes corner numbers as an input source. IMO, event notes are the best choice here, as pointed out by @harningle already.

**Comment by MrBiscuit921** [comment]
@theOehrly yeah i think either event notes or [f1 website](https://www.formula1.com/en/racing/2024/united-arab-emirates/circuit)
